### PR TITLE
[ASM] add load for asm

### PR DESCRIPTION
**Pull request by anna-git**
Add load for asm

### CODE REVIEW

**Review by e-n-0 — APPROVED** [review]
LGTM

File: JunkyardLoad/JunkyardLoad.cs [hunk 1]
```diff
@@ -33,6 +37,8 @@ public static async Task JunkyardNetcore31Metrics([TimerTrigger(MetricsInterval)
             await GetMetrics("dd-dotnet-linux-latest-build-stats", service: "net8-linux-latest-build-stats", log);
             await GetMetrics("dd-dotnet-linux-latest-build-profiler-default", service: "net8-linux-latest-build-profiler-default", log);
             await GetMetrics("dd-dotnet-linux-latest-build-profiler-all", service: "net8-linux-latest-build-profiler-all", log);
+            await GetMetrics("dd-dotnet-linux-latest-build-profiler-all", service: "net8-linux-latest-build-profiler-all", log);
```
Comment (andrewlock) [inline JunkyardLoad/JunkyardLoad.cs:40]: I think this line is a duplicate? 🤔 

```suggestion
```

File: JunkyardLoad/JunkyardLoad.cs [hunk 2]
```diff
@@ -107,6 +113,12 @@ public static async Task JunkyardNet8LinuxDevProfilerAll([TimerTrigger(LoadTestI
             await JunkyardDump("dd-dotnet-linux-latest-build-profiler-all", log: log, service: "net8-linux-latest-build-profiler-all");
         }
 
+        [FunctionName("dd-dotnet-security-aspnetcore")]
```
Comment (andrewlock) [inline JunkyardLoad/JunkyardLoad.cs:116]: You added a constant for this 😄 

```suggestion
        [FunctionName(SecurityAppUrlPrefix)]
```

**Review by andrewlock — COMMENTED** [review]
Thanks!

**Pull request merged**